CMakeLists.txt

```diff
@@ -101,7 +101,6 @@ include(AzureVersion)
 
 if(BUILD_SAMPLES)
   add_subdirectory(samples/helpers/get-env)
-  add_subdirectory(samples/helpers/service)
```

Comment:
Let's try to find the other way... Basically, what we have is header-only library, let's fix cmake so that it works on linux too. I would really like to avoid the `..\..\..`, and make it look like any other azure-sdk library. Even at a cost of it having a .cpp file.

Reply:
Given that `service` is supposed to be a placeholder for `azure-anyservice`, I thikn it is better to add a .cpp file. We can make it an `OBJECT` library similar to `get-env`, consuming it is semantically the same as regular library, unlike `INTERFACE` library.

Reply:
@vhvb1989, how about this? - #3728